awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -9,7 +9,7 @@ import CardCloseButton from '@components/CardCloseButton';
 import ContentError from '@components/ContentError';
```

Comment:
This is ready to be refactored into hooks, which I can/plan to do. However, I wanted to keep the scope of this PR smaller.

Reply:
👍 JT Form also needs functionalized.  I started doing it when I was moving thigns to the new useField formik hook, but it started getting a lot bigger, so for similar reasons, I decided to hold off on that.